force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -19,77 +19,14 @@ Let's dive into the specifics of each operation:
 */
 trigger AccountTrigger on Account (before insert, after insert) {
 
-    /*
-    * Account Trigger
-    * When an account is inserted change the account type to 'Prospect' if there is no value in the type field.
-    * Trigger should only fire on insert.
-    */
     if (Trigger.isBefore && Trigger.isInsert) {
-        for (Account acc : Trigger.new) {
-            if (acc.Type == null) {
-                acc.Type = 'Prospect';
-            }
-        }
+        AccountHelper.setTypeProspect(Trigger.new);
+        AccountHelper.addressCopy(Trigger.new);
+        AccountHelper.setRating(Trigger.new);
```

Comment:
Good job combining these together into 1 if statement because they are the same. Makes it easier to trace when they are all together